front/pages/api/apps/[user]/[sId]/runs/index.ts

```diff
@@ -108,8 +122,11 @@ export default async function handler(req, res) {
           });
 
           try {
-            for await (const chunk of streamRes.body) {
+            for await (const chunk of streamChunks(streamRes.body)) {
```

Comment:
TS wouldn't let me iterate directly on `streamRes.body`

Reply:
I looked into it, I could not find a better answer.
Looks like some new api maybe? Not sure.